mobile/components/views/Attractions/Router.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from "react";
+import {DrawerItem} from '@react-navigation/drawer';
+import { View } from "react-native";
+
+
+
+const AttractionSubMenu: React.FC<any> = (props) => {
```

Comment:
Please add some kind of comment/notification that we will need to add types here  in the future